### PR TITLE
Add bitarray recipe

### PR DESCRIPTION
**Pull request by Novfensec**
Contains a recipe for [bitarray](https://github.com/ilanschnell/bitarray).
Tested and built an apk sucessfully at: https://github.com/Novfensec/AprsChat/actions/workflows/buildozer_action.yml

### CODE REVIEW

File: pythonforandroid/recipes/bitarray/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from pythonforandroid.recipe import CppCompiledComponentsPythonRecipe
+
+
+class BitarrayRecipe(CppCompiledComponentsPythonRecipe):
+    stl_lib_name = "c++_shared"
+    version = "3.0.0"
+    url = "https://github.com/ilanschnell/bitarray/archive/refs/tags/3.0.0.tar.gz"
```
Comment (AndreMiras) [inline pythonforandroid/recipes/bitarray/__init__.py]:
```suggestion
    url = "https://github.com/ilanschnell/bitarray/archive/refs/tags/{version}.tar.gz"
```

**Review by AndreMiras — CHANGES_REQUESTED** [review]
Thanks for contributing.
Because we can see if it compile on the CI, can you update the `url` member to include the version placeholder?

**Comment by Novfensec** [comment]
> Thanks for contributing.
> Because we can see if it compile on the CI, can you update the `url` member to include the version placeholder?
Added `{version}` placeholder.

**Review by Novfensec — COMMENTED** [review]
Added `{version}` placeholder.

**Review by AndreMiras — APPROVED** [review]
LGTM thanks

**Pull request merged**